.github/workflows/ci.yml

```diff
@@ -69,8 +69,10 @@ jobs:
         run: composer update --no-progress --no-interaction --prefer-dist --prefer-lowest
         if: ${{ matrix.dependencies == 'lowest' }}
 
-      - name: Run tests
-        run: vendor/bin/phpunit --coverage-clover=coverage.xml
+      - name: Run unit tests
+        run: vendor/bin/phpunit --testsuite unit --coverage-clover=coverage.xml
```

Comment:
I don't understand the advantages of splitting the test suite in two, except perhaps being able to run them independently (but that is rare) 🤔

Reply:
This is because the OOM tests do not run correctly because the code coverage code causes them to OOM again. So I disabled the coverage for the OOM tests specifically now and added all other `phpt` tests back to the normal test suite with coverage.

When running `phpunit` locally all tests in all suites are executed by default so there is no difference for local development.